config/settings/base.py

```diff
@@ -201,6 +201,10 @@
     "care.audit_log.middleware.AuditLogMiddleware",
 ]
 
+# add RequestTimeLoggingMiddleware based on the environment variable
+if env.bool("ENABLE_REQUEST_TIME_LOGGING", default=True):
+    MIDDLEWARE.append("config.middleware.RequestTimeLoggingMiddleware")
```

Comment:
```suggestion
    MIDDLEWARE.insert(0, "config.middleware.RequestTimeLoggingMiddleware")
```

adding it to the [top](https://docs.djangoproject.com/en/dev/topics/http/middleware/#middleware-order-and-layering) will show us the time spent in other middlewares

cc: @vigneshhari